internal/docker/supported.go

```diff
@@ -81,5 +96,30 @@ func cleanupDockerResource(pool *dockertest.Pool, resource *dockertest.Resource)
 	if strings.Contains(err.Error(), "No such container") {
 		return nil
 	}
-	return fmt.Errorf("Failed to cleanup local container: %s", err)
+	return fmt.Errorf("failed to cleanup local container: %s", err)
+}
+
+// splitImage takes the WithDatabaseImage option and separates
+// it into repo + tag. If a tag is not found, it sets the tag to latest
+func splitImage(opts Options) (string, string, error) {
+
+	separated := strings.Split(opts.withContainerImage, ":")
+	separatedlen := len(separated)
+
+	switch separatedlen {
+	case 1:
+		if separated[0] == "postgres" {
+			return separated[0], "11", nil
+		}
+		return "", "", fmt.Errorf("valid reference format is repo:tag, if"+
```

Comment:
I am not exactly sure what you want this to do, it seems like this is the correct behavior where `postgres` becomes `postgres:11` otherwise it fails.  However, an alternative is to do what the comment says it will do here:
```
if separated[0] == "postgres" {
	return separated[0], "11", nil
}
return separated[0], "latest", nil
```
It is fine to leave as is as well, mainly pointing out discrepancy between the comment and code...

Reply:
Or 
```
version := "latest"
if separated[0] == "postgres" {
	version = "11"
}
return separated[0], version, nil
```